src/test/java/org/primeframework/mvc/parameter/NamedParameterHandlerTest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2023, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc.parameter;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.primeframework.mvc.PrimeException;
+import org.primeframework.mvc.action.annotation.Action;
+import org.primeframework.mvc.action.config.ActionConfigurationBuilder;
+import org.primeframework.mvc.action.config.DefaultActionConfigurationBuilder;
+import org.primeframework.mvc.content.binary.BinaryActionConfigurator;
+import org.primeframework.mvc.content.json.JacksonActionConfigurator;
+import org.primeframework.mvc.parameter.annotation.NamedParameter;
+import org.primeframework.mvc.util.DefaultURIBuilder;
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+/**
+ * @author Lyle Schemmerling
+ */
+public class NamedParameterHandlerTest {
+  @Test
+  public void test() {
+    var builder = new DefaultActionConfigurationBuilder(new DefaultURIBuilder(), new HashSet<>(Arrays.asList(new JacksonActionConfigurator(), new BinaryActionConfigurator())));
+    buildExpectException(builder, InvalidGetterAction.class, "The action class [InvalidGetterAction] has a NamedParameter annotated method [foo] for name [getme] which is not a valid java bean property method.");
+
+    buildExpectException(builder, InvalidSetterAction.class, "The action class [InvalidSetterAction] has a NamedParameter annotated method [bar] for name [setme] which is not a valid java bean property method.");
+
+    buildExpectException(builder, PropClashWithFieldAction.class, "The action class [PropClashWithFieldAction] has a NamedParameter annotated method [setBar] which collides with defined field with name [target].");
```

Comment:
I think this error has to contain the name so it is clear from the error why it it causes a name collision. 

Suggestion:

> "The action class [PropClashWithFieldAction] has a method [setBar] annotated with NamedParameter for name [target] collides with a defined field named [target]."